plasma_framework/contracts/src/exits/utils/MoreVpFinalization.sol

```diff
@@ -21,26 +21,27 @@ library MoreVpFinalization {
     function isStandardFinalized(
         PlasmaFramework framework,
         bytes memory txBytes,
-        TxPosLib.TxPos memory txPos,
+        PosLib.Position memory txPos,
         bytes memory inclusionProof
     )
         internal
         view
         returns (bool)
     {
+        require(txPos.outputIndex == 0, "Invalid transaction position");
```

Comment:
Why do require outputIndex to be zero?

Reply:
🤔and no test on CI failed for this?

Reply:
@thec00n because it previously was a TxPos. I left it as it was, we still require it to be a tx pos (utxo pos of 0 index output)